src/Umbraco.Core/Dictionary/UmbracoCultureDictionary.cs

```diff
@@ -47,9 +48,9 @@ public DefaultCultureDictionary(CultureInfo specificCulture, ILocalizationServic
     }
 
     /// <summary>
-    ///     Returns the current culture
+    ///     Returns the defualt umbraco's back office culture
     /// </summary>
-    public CultureInfo Culture => _specificCulture ?? Thread.CurrentThread.CurrentUICulture;
+    public CultureInfo Culture => _specificCulture ?? new CultureInfo( _localizationService.GetDefaultLanguageIsoCode());
```

Comment:
I'm afraid we can't really introduce this change in minor since this is behaviorally breaking, unfortunately. Since this could suddenly change the culture of people's `UmbracoCultureDictionary`, potentially breaking the code. 

However, I do agree that it would make more sense to default to the default language instead, so it may be one for a major 😄

Reply:
hello , thanks for replying , 
honestly I don't remember what is this PR for , so I give you full liberty over it. tweak or change what ever u like

Reply:
Yeah, it's been quite a while, sorry about that, but I'll go ahead and undo this specific part, and then move that to V13. then we can merge the rest of this PR 😄